### ISSUE TITLE
Quotes in node-sass

### ISSUE THREAD

**Issue by joneff** [sass/node-sass#282]
Note: I originally posted this issue in madskristensen/WebEssentials2013#880. Apparently it was not related to WebEssentials but is instead related to either node-sass or libsass. If the issue is not related to node-sass I will forward it to hcatlin/libsass

---

Quotes behaviour is rather strange node-sass -- at times they are added at times removed. Examine the following code

``` scss
$q: unquote('"');
$p1: unquote("(");
$p2: unquote(")");
$c: "something";

body {
    test: $p1 + $q + $p2;
    test: $q + $p1 + $p2;
    test: unquote('"') + asd;
}
```

if I run it against sass gem I get the expected output:

``` css
body {
  test: (");
  test: "();
  test: "asd; }
```

where as with node-sass i get:

``` css
body {
  test: ();
  test: "()";
  test: "asd"; }
```

So to sum up when doing string + string
- quotes in the begging of the line are preserved
- if an end quote is missing, it's added
- if there is no quote in the beginning, they are stripped
